src/main/java/edu/kis/vh/nursery/storage/IntStorageInterface.java

```diff
@@ -0,0 +1,18 @@
+package edu.kis.vh.nursery.storage;
+
+public interface IntStorageInterface {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.